Assets/MixedRealityToolkit.Examples/Demos/EyeTracking/DemoTargetSelections/Prefabs/TargetType (1).prefab

```diff
@@ -4895,4 +4895,4 @@ MonoBehaviour:
   m_Name: 
   m_EditorClassIdentifier: 
   RotateByEulerAngles: {x: 0, y: 1, z: 1}
-  speed: 1
+  speed: 0.3
```

Comment:
Was this change intentional?

Reply:
The change was intentional, however, I was meaning to submit it in a separate patch. I noticed that the gems in the eye tracking scene rotate VERY was what moving the cursor over them. This is not the expected behavior. Not sure what changed. I can revert it for now until looking into whether the fast behavior is only in the Editor or also on the device.